src/diffusers/models/attention_processor.py

```diff
@@ -1416,6 +1474,7 @@ def __call__(self, attn: "Attention", hidden_states, encoder_hidden_states=None,
     AttnAddedKVProcessor2_0,
     LoRAAttnProcessor,
     LoRAXFormersAttnProcessor,
+    TuneAVideoAttnProcessor,
```

Comment:
I think this attention processor is specific to the Tune a Video pipeline, no? 

If so, it might be better to add it to the pipeline class module itself like:

https://github.com/huggingface/diffusers/blob/5559d0423771e8b6e454b1541164e0d5d54b6265/src/diffusers/pipelines/text_to_video_synthesis/pipeline_text_to_video_zero.py#L39

WDYT?

Cc: @patrickvonplaten

Reply:
I'm ok with moving it to the pipeline class as it's specific to the pipeline.

Reply:
Moved the processor to the pipeline class. init is a bit hacky because only one of the attentions needs to use the custom processor. Let me know what you think:

https://github.com/Abhinay1997/diffusers/blob/tune_a_video_port/src/diffusers/pipelines/tune_a_video/pipeline_tune_a_video.py#L181